dandischema/models.py

```diff
@@ -1815,6 +1867,12 @@ class Asset(BareAsset):
         json_schema_extra={"readOnly": True, "nskey": "schema"}
     )
```

Comment:
look here for an example on how we add extra to json schema.

May be you could add extra for your `@context` which would state something like `"includeInUI": False`